libs/features/components/ui-lists/elewa-group-vertical-list-one/elewa-group-vertical-list-one.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-elewa-group-vertical-list-one',
+  templateUrl: './elewa-group-vertical-list-one.component.html',
+  styleUrls: ['./elewa-group-vertical-list-one.component.css']
+})
+export class ElewaGroupVerticalListOneComponent {
+  articles: any
```

Comment:
This entire component needs to be on a library, check the issue description again.